source/_integrations/frontier_silicon.markdown

```diff
@@ -4,7 +4,7 @@ description: Instructions on how to integrate Frontier Silicon Internet Radios i
 ha_category:
   - Media Player
 ha_iot_class: Local Push
-ha_release: '0.40'
+ha_release: '2022.1'
```

Comment:
This version marks the introduction (not the last update).
Please revert this change.